packages/eas-cli/src/utils/analytics.ts

```diff
@@ -0,0 +1,65 @@
+import { logEvent } from '../analytics';
+
+export type TrackingContext = Record<string, string | number | boolean>;
+
+export enum SubmissionEvent {
+  SUBMIT_COMMAND = 'submit cli submit command',
+  SUBMIT_COMMAND_SUCCESS = 'submit cli success',
+  SUBMIT_COMMAND_FAIL = 'submit cli fail',
+  GATHER_CREDENTIALS_SUCCESS = 'submit cli gather credentials success',
+  GATHER_CREDENTIALS_FAIL = 'submit cli gather credentials fail',
+  GATHER_ARCHIVE_SUCCESS = 'submit cli gather archive success',
+  GATHER_ARCHIVE_FAIL = 'submit cli gather archive fail',
+  MUTATION_SUCCESS = 'submit cli server mutation success',
```

Comment:
maybe SUBMIT_REQUEST_SUCCESS ? (the fact that it's graphql mutation is not that important)